### ISSUE TITLE
[0.8.1] GOG version Exe check failed 0x62952F34

### ISSUE THREAD

**Issue by A9G-Data-Droid** [Nibre/MotherVR#237]
Similar to what happened with the Epic version, the GOG version fails the Exe check.

![image](https://user-images.githubusercontent.com/26984068/172278936-4908f0cc-ec6d-4d6a-8596-a9353b7b27d1.png)

**Comment by Mobeeuz** [comment]
Yes, hell has frozen over and we have a DRM-free GOG release - please update :)

**Comment by A9G-Data-Droid** [comment]
Or release source, like stated in the last release note. There is enough activity here that someone will take a crack at it.

**Comment by githububble** [comment]
Great, so I just wasted my money. ☹☹☹ I only bought it to play in vr. Why doesn't it work? It's the same game, really upset me...

**Comment by A9G-Data-Droid** [comment]
> Great, so I just wasted my money. ☹☹☹ I only bought it to play in vr. Why doesn't it work? It's the same game, really upset me...

The reason is that it uses static memory address patching to function. Any change to the binary could modify the location of memory addresses so the software checks to see if the EXE has a known CRC. If the CRC check fails it doesn't proceed, knowing that modifying the wrong memory address could be disastrous.

**Comment by githububble** [comment]
> > Great, so I just wasted my money. ☹☹☹ I only bought it to play in vr. Why doesn't it work? It's the same game, really upset me...
> 
> The reason is that it uses static memory address patching to function. Any change to the binary could modify the location of memory addresses so the software checks to see if the EXE has a known CRC. If the CRC check fails it doesn't proceed, knowing that modifying the wrong memory address could be disastrous.

Seeing as [Nibre](https://github.com/Nibre) has no activity recently, I've asked for a refund of the game. Hopefully it goes on sale soon, as it's really expensive for an old game and its been given away free a bunch which is just annoying as most people will not know about that. I had this game when it came out but for whatever reason it's not in any of my accounts anymore.

**Comment by A9G-Data-Droid** [comment]
He is expected to come back when he is done working on his Halo VR project. Life happens. I know I don't have as much time for open source as I would like.

**Comment by Mobeeuz** [comment]
> He is expected to come back when he is done working on his Halo VR project..

Nice to see that didn't die...

**Comment by Mobeeuz** [comment]
> Seeing as [Nibre](https://github.com/Nibre) has no activity recently, I've asked for a refund of the game. Hopefully it goes on sale soon, as it's really expensive for an old game

It's %75 off on GOG right now, how much bigger of a sale did you want?

**Comment by A9G-Data-Droid** [comment]
> > He is expected to come back when he is done working on his Halo VR project..
> 
> Nice to see that didn't die...

His twitter, that he advertises everywhere, hasn't posted in over a year. I don't know that he is alive...

**Comment by githububble** [comment]
> > Seeing as [Nibre](https://github.com/Nibre) has no activity recently, I've asked for a refund of the game. Hopefully it goes on sale soon, as it's really expensive for an old game
> 
> It's %75 off on GOG right now, how much bigger of a sale did you want?

It's about the mod not working with the gog version, I refunded it as I wanted to play it in VR.. The context is pretty obvious. On steam and epic it's £35. Thats a lot for an 8 year old game that I've already played.

**Comment by githububble** [comment]
It's on steam summer sale now, just incase anyone looking at this. I look forward to finally trying it in vr!

**Comment by georgeduckett** [comment]
@githububble FYI the steam version doesn't work in VR in Windows 11 (Epic version does though).

**Comment by Espionage724** [comment]
> @githububble FYI the steam version doesn't work in VR in Windows 11 (Epic version does though).

https://github.com/Nibre/MotherVR/issues/229 for anyone wanting to follow this

**Comment by matrushko** [comment]
whaaaaaaaaaaat?

I was sent here from telegram, telling me "look for it in Reddit, it seems to be a solution there" but I see THERE IS NOT!

buaaaaaaa, Couldn't I play this game in w11 in VR then?

ffffff****

**Comment by Subtly-Dismiss** [comment]
Anybody landing here - you can fix Win11 by moving the game to a folder and labelling it "AlienIsolation" - No Space, then creating a symlink to the Steam directory.

https://forums.overclockers.co.uk/threads/alien-isolation.18570957/page-126#post-36061491

GOG version still does not work!!!
